Data/Generated.lua

```diff
@@ -103,3 +103,120 @@ for index, name in ipairs(keystones) do
 end
 table.insert(skinOfTheLords, "Corrupted")
 table.insert(data.uniques.generated, table.concat(skinOfTheLords, "\n"))
+
+--[[ 3 scenarios exist for legacy mods
+	- Mod changed, but kept the same mod Id
+		-- Has legacyMod
+	- Mod removed, or changed with a new mod Id
+		-- Has only a version when it changed
+	- Mod changed/removed, but isn't legacy
+		-- Has empty table to exclude it from the list
+]]
+local watchersEyeLegacyMods = {
+	["ClarityManaAddedAsEnergyShield"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(12-18)")) end,
+	},
+	["ClarityReducedManaCost"] = {
+		["version"] = "3.8.0",
+	},
+	["ClarityManaRecoveryRate"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(20-30)")) end,
+	},
+	["DisciplineEnergyShieldRecoveryRate"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(20-30)")) end,
+	},
+	["MalevolenceDamageOverTimeMultiplier"] = {
+		["version"] = "3.8.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(36-44)")) end,
+	},
+	["MalevolenceLifeAndEnergyShieldRecoveryRate"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(15-20)")) end,
+	},
+	["PrecisionIncreasedCriticalStrikeMultiplier"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(30-50)")) end,
+	},
+	["VitalityDamageLifeLeech"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(1-1.5)")) end,
+	},
+	["VitalityFlatLifeRegen"] = {
+		["version"] = "3.12.0",
+	},
+	["VitalityLifeRecoveryRate"] = {
+		["version"] = "3.12.0",
+		["legacyMod"] = function(currentMod) return (currentMod:gsub("%(.*%)", "(20-30)")) end,
+	},
+	["WrathLightningDamageManaLeech"] = {
+		["version"] = "3.8.0",
+	},
+	["PurityOfFireReducedReflectedFireDamage"] = { },
+	["PurityOfIceReducedReflectedColdDamage"] = { },
+	["PurityOfLightningReducedReflectedLightningDamage"] = { },
+	["MalevolenceSkillEffectDuration"] = { },
+	["ZealotryMaximumEnergyShieldPerSecondToMaximumEnergyShieldLeechRate"] = { },
+	["MalevolenceColdDamageOverTimeMultiplier"] = { },
+	["MalevolenceChaosNonAilmentDamageOverTimeMultiplier"] = { },
+}
+
+local watchersEye = { [[
+		Watcher's Eye
+		Prismatic Jewel
+		Source: Drops from unique{The Elder}
+		Has Alt Variant: true
+		Has Alt Variant Two: true]]
+}
```

Comment:
```suggestion
local watchersEye = {
[[
Watcher's Eye
Prismatic Jewel
Source: Drops from unique{The Elder}
Has Alt Variant: true
Has Alt Variant Two: true
]]
}
```
This formatting is more in line with the existing uniques.